MontyTRC89/TombEngine
# Animation refactors tier 2

Tier 2 of around 5.
Introduces new `SetAnimation()` overload which allows for easy setting of animations from different slots.

To test:
- All vehicles (mounts, dismounts, deaths).
- Special creature deaths (TR1 torso boss, shiva, etc.).
- Any other special animations.
- Try killing various creature entities and see if they crash the game.
